infra/network/src/test/java/com/simprints/infra/network/url/BaseUrlProviderImplTest.kt

```diff
@@ -14,7 +14,10 @@ import org.junit.Test
 class BaseUrlProviderImplTest {
 
     companion object {
+        private const val URL_SUFFIX = "/androidapi/v2/"
+
         private const val URL = "https://test"
+        private const val URL_WITH_PREFIX = "https://test$URL_SUFFIX"
```

Comment:
Was the intended name `URL_WITH_SUFFIX`?

Reply:
Yes, fixed.